packages/common/http/src/resource.ts

```diff
@@ -29,6 +29,7 @@ import {HttpEventType, HttpProgressEvent, HttpResponseBase} from './response';
 import {HttpHeaders} from './headers';
 import {HttpParams} from './params';
 import {HttpResourceRef, HttpResourceOptions, HttpResourceRequest} from './resource_api';
+import {encapsulateResourceError} from '../../../core/src/resource/resource';
```

Comment:
```suggestion
import {encapsulateResourceError} from '@angular/core';
```
This is an external entry, the import shouldn't be relative

Reply:
This function was very simple that I declared it independently in this file resulting in  no need to import it.